connectivity/libraries/nanostack-libservice/test/libService/unittest/nsnvmhelper/main.cpp

```diff
@@ -1,5 +1,18 @@
-/*
- * Copyright (c) 2015 ARM Limited. All rights reserved.
+/* mbed Microcontroller Library
```

Comment:
```suggestion
/*
```
The first line can be left empty - the text is there in some files for historical reasons only.

Reply:
I would also remove this first line as it uses old name that is not correct anymore.